lib/email_address.rb

```diff
@@ -13,59 +13,28 @@ module EmailAddress
     require "email_address/canonical_email_address_type"
   end
 
+  class << self
+    %i(valid? error normal redact munge canonical reference).each do |proxy_method|
```

Comment:
[`%i[]` is better](https://github.com/bbatsov/ruby-style-guide#percent-literal-braces) (because it's array :man_shrugging:).